### PR TITLE
Execute account recovery flow - request code

### PR DESCRIPTION
**Pull request by tung2744**
ref #3370

### CODE REVIEW

**Comment by tung2744** [comment]
@louischan-oursky Ready for review, thanks!

File: pkg/lib/authenticationflow/declarative/intent_request_account_recovery_flow.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	case len(flows.Nearest.Nodes) == 0:
+		return authflow.NewSubFlow(&IntentRequestAccountRecoveryFlowSteps{
+			JSONPointer: i.JSONPointer,
+		}), nil
+	}
+
+	return nil, authflow.ErrIncompatibleInput
+}
+
+func (i *IntentRequestAccountRecoveryFlow) GetEffects(ctx context.Context, deps *authflow.Dependencies, flows authflow.Flows) ([]authflow.Effect, error) {
```
Comment (louischan-oursky) [inline pkg/lib/authenticationflow/declarative/intent_request_account_recovery_flow.go]: We can skip conforming to EffectGetter if we do not have any effects to perform.

File: pkg/lib/config/authentication_flow_test.go [hunk 2]
```diff
@@ -193,3 +193,54 @@ reauth_flows:
...
+			err = json.Unmarshal([]byte(inputJSON), &input)
+			So(err, ShouldBeNil)
+
+			encodedCfg, err := json.Marshal(cfg)
+			So(err, ShouldBeNil)
+
+			encodedInput, err := json.Marshal(input)
+			So(err, ShouldBeNil)
+
+			// NOTE(tung): Disable the schema test for now because it was removed from config schema
```
Comment (louischan-oursky) [inline pkg/lib/config/authentication_flow_test.go:221]: Do we need to fix the notes in this file in this PR?

Comment (tung2744) [reply pkg/lib/config/authentication_flow_test.go:221]: @louischan-oursky If we don't make the flow configurable at the moment, I think we don't need to fix it.

**Pull request merged**